src/content/docs/synthetics/synthetic-monitoring/using-monitors/manage-monitor-runtimes.mdx

```diff
@@ -46,7 +46,7 @@ To see a history of monitor version upgrades, query the [`NrAuditEvent`](/docs/s
 
 Make your scripted monitors more contextually aware by using the `$env` variable properties. When the script executes, these properties represent important information about the runtime environment.
 
-You do not need to import `$env` because it is readily available, similar to the `$browser` and `$http` variables. For example:
+You don't need to import `$env` because it's readily available, similar to the `$browser` and `$http` variables. For example:
```

Comment:
This sentence is a little confusing, I'm not sure what "readily available" means versus not "readily available", maybe "already available" is better?